docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
         - ALPINE_RUBY_VERSION
     # mounting . is causing seg-fault on macosx
     #volumes:
-      #- .:/app
+    #- .:/app
     depends_on:
```

Comment:
If we're moving to the newer conventions, they changed the recommended filename from "docker-compose" to just "compose" 

https://docs.docker.com/compose/gettingstarted/#step-2-define-services-in-a-compose-file

Reply:
This PR is not about conventions, but that they have completely removed `docker-compose` from this image.

Reply:
I created https://github.com/projectblacklight/blacklight/issues/3256 to track this suggestion.